packages/backend-common/package.json

```diff
@@ -42,6 +44,7 @@
     "@types/http-errors": "^1.6.3",
     "@types/morgan": "^1.9.0",
     "@types/supertest": "^2.0.8",
+    "@types/webpack-env": "^1.15.2",
```

Comment:
Should we move all @types here to the dependencies so they can be used by backend-plugins?

Reply:
We want to add this as a dependency of the cli afaik, and add it to the list of types in the common tsconfig. One of the outcomes of this change should be that we remove all the tsconfigs in the backend packages.

Reply:
Yes, but I'd rather have it as a separate PR, wdyt?